tests/www/api/experimental/test_dag_runs_endpoint.py

```diff
@@ -27,10 +28,26 @@
 from tests.test_utils.decorators import dont_initialize_flask_app_submodules
 
 
-class TestDagRunsEndpoint(unittest.TestCase):
-    @classmethod
-    def setUpClass(cls):
-        super().setUpClass()
+@pytest.fixture(scope="module")
```

Comment:
My fear is that the module never closes, so this creates a side effect between all tests. Am I right?

Reply:
My understanding is, the fixuture’s lifeline is not tied to the module *itself*, but pytest internally tracks whether all tests in the module and tears down the fixture after they all run. `scope` means more like “tests in this scope” in the lexical sense, not the actual function/class/module the fixture lives in.

Reply:
Yeah, this is not tied to the module being imported/unimported, but to what tests Pytest has left to run.